iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothubservices/FileUploadTests.java

```diff
@@ -274,11 +273,13 @@ public void uploadToBlobAsyncSingleFileZeroLength() throws URISyntaxException, I
         // act
         deviceClient.uploadToBlobAsync(fileUploadState[0].blobName, fileUploadState[0].fileInputStream, fileUploadState[0].fileLength, new FileUploadCallback(), fileUploadState[0]);
 
-        FileUploadNotification fileUploadNotification = getFileUploadNotificationForThisDevice(scDevice);
-
         // assert
-        verifyNotification(fileUploadNotification, fileUploadState[0]);
-        assertTrue(buildExceptionMessage("File upload callback was not triggered", deviceClient), fileUploadState[0].isCallBackTriggered);
+        if (!isBasicTierHub)
+        {
+            FileUploadNotification fileUploadNotification = getFileUploadNotificationForThisDevice(scDevice);
```

Comment:
>FileUploadNotification fileUploadNotification = getFileUploadNotificationForThisDevice(scDevice); [](start = 12, length = 97)

are we allowed to just ignore the fileupload notification for standard tier as well? Why are we only ignoring it for basic tier?

Reply:
It was agreed upon with Priyanka on basic tier ONLY so that we only use 1 hub to test for all platforms on canary

---
In reply to: [274584499](https://github.com/Azure/azure-iot-sdk-java/pull/485#discussion_r274584499) [](ancestors = 274584499)